package.json

```diff
@@ -1,11 +1,38 @@
 {
+  "name": "athens",
+  "author": "athensresearch",
+  "version": "1.0.0-beta.2",
+  "description": "Open-Source Networked Thought",
+  "main": "resources/main.js",
+  "build": {
+    "generateUpdatesFilesForAllChannels": true,
+    "linux": {
+      "target": [
+        "AppImage"
+      ]
+    },
+    "publish": {
+      "provider": "s3",
+      "bucket": "athens-apps",
+      "region": "us-east-2"
+    }
+  },
+  "scripts": {
+    "dev": "./node_modules/.bin/shadow-cljs watch main renderer",
```

Comment:
What is the difference between `renderer` and `app` builds? Why do we sometimes use `package.json` and sometimes `project.clj` for scripts?

Reply:
Keeping both for now, can handle redundancy in `project.clj`, `shadow-cljs.edn`, `package.json`, and `scripts/` in another issue.